docs/blog/posts/multi-stage-delivery-using-gitops.md

```diff
@@ -0,0 +1,367 @@
+---
+date: 2024-03-26
+authors: [bacherfl]
+description: >
+  This blog post explains how to use the promotion phase of Keptn to implement a multi stage delivery GitOps workflow.
+categories:
+  - GitOps
+  - Observability
+comments: true
+---
+
+# Multi Stage Delivery using GitOps
+
+In multi-stage environments it can often become a challenge to keep an overview of
+how a particular version of a workload progresses through different stages.
+This can make it difficult to trace back which exact introduced a problem
+when something goes wrong in one of the deployment stages.
```

Comment:
```suggestion
This can make it difficult to precisely trace exactly which modification introduced a problem
when something goes wrong in one of the deployment stages.
```